### PR TITLE
fix #2470 track helpshift replies

### LINKED ISSUE #2470: Adding tracking for when a user replies to Helpshift
WordPress-iOS Ref https://github.com/wordpress-mobile/WordPress-iOS/pull/3463

Helpshift now permits to listen to different events happening in their lib - SDK doc: https://www.helpshift.com/docs/api/android/v3.x/reference/com/helpshift/Helpshift.HelpshiftDelegate.html

### PR DESCRIPTION
**Pull request by maxme**
fix #2470 track helpshift replies

### CODE REVIEW

**Comment by daniloercoli** [comment]
The code for the `Tracks` tracker is missing in this PR. We should add the code to both trackers, Mixpanel and Tracks, since Mixpanel will be sunseted shortly.

**Comment by maxme** [comment]
> The code for the Tracks tracker is missing in this PR. We should add the code to both trackers, Mixpanel and Tracks, since Mixpanel will be sunseted shortly.

Fixed in b5da88c

File: libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java [hunk 1]
```diff
@@ -286,6 +286,9 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
             case SUPPORT_OPENED_HELPSHIFT_SCREEN:
                 eventName = "support_helpshift_screen_opened";
                 break;
+            case SUPPORT_SENT_REPLY_TO_SUPPORT_MESSAGE:
+                eventName = "support_sent_reply_to_support_message";
```
Comment (daniloercoli) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: Could you please change `support_sent_reply_to_support_message` to `support_reply_to_support_message_sent` ?

Comment (maxme) [inline libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java]: renamed in f644083

**Comment by daniloercoli** [comment]
:shipit:

**Pull request merged**